Hmm, something wrong with mock_requests. It doesn't even work for the async openfaas call (which I did not modify):

`
3170
  >       raise exceptions.NoMockAddress(request)
3171
  E       requests_mock.exceptions.NoMockAddress: No mock address: POST http://open-faas.io/async-function/function_name
3172
  
3173
  /usr/local/lib/python3.6/site-packages/requests_mock/adapter.py:258: NoMockAddress`

--- a/tests/providers/openfaas/hooks/test_openfaas.py
+++ b/tests/providers/openfaas/hooks/test_openfaas.py
@@ -101,7 +101,7 @@ class TestOpenFaasHook(unittest.TestCase):
         mock_get_connection.return_value = mock_connection
 
         with self.assertRaises(AirflowException) as context:
-            self.hook.invoke_async_function({})
+            self.hook.invoke_function({})
         self.assertIn('failed to invoke function', str(context.exception))
 
     @mock.patch.object(BaseHook, 'get_connection')
@@ -110,11 +110,11 @@ class TestOpenFaasHook(unittest.TestCase):
         m.post(
             "http://open-faas.io" + self.INVOKE_FUNCTION + FUNCTION_NAME,
             json=self.mock_response,
-            status_code=202,
+            status_code=200,
         )
         mock_connection = Connection(host="http://open-faas.io")
         mock_get_connection.return_value = mock_connection
-        self.assertEqual(self.hook.invoke_async_function({}), None)
+        self.assertEqual(self.hook.invoke_function({}), None)
 
     @mock.patch.object(BaseHook, 'get_connection')
     @requests_mock.mock()

